<launch>

    <node pkg="planning" exec="planning_manager_node" name="planning_manager_node" output="screen">
        <remap from="planning/lanelet2_planner/trajectory" to="planning/global_planner/trajectory"/>
        <remap from="planning/lattice_planner/trajectory" to="planning/local_planner/trajectory"/>
        <param name="local_planning_frequency" value="5"/>
    </node>

    <include file="$(find-pkg-share control)/launch/control.launch"/>

    <include file="$(find-pkg-share carla_bridge)/launch/carla_bridge.launch"/>

</launch>
